view-generator-framework/build.gradle.kts

```diff
@@ -44,7 +44,6 @@ dependencies {
   // Needed for flink metric exporter. Used for Hypertrace and debugging.
   runtimeOnly("org.apache.flink:flink-metrics-slf4j:1.10.1")
 
-  testImplementation("org.apache.kafka:kafka-clients:5.5.0-ccs")
```

Comment:
@sanjay-nagaraj See if we can also remove repositories in L9

Reply:
I cant.  > Could not find io.confluent:kafka-avro-serializer:5.5.0.
     Searched in the following locations:
       - file:/Users/sanjay/.m2/repository/io/confluent/kafka-avro-serializer/5.5.0/kafka-avro-serializer-5.5.0.pom
       - https://repo.maven.apache.org/maven2/io/confluent/kafka-avro-serializer/5.5.0/kafka-avro-serializer-5.5.0.pom
       - https://dl.bintray.com/hypertrace/maven/io/confluent/kafka-avro-serializer/5.5.0/kafka-avro-serializer-5.5.0.pom
     Required by:
         project :view-generator-framework > org.hypertrace.core.flinkutils:flink-utils:0.1.1

Reply:
Haha. This is probably because in flink utils it includes it as an api dependency. Here's it's an implementation. That's why transitivity doesn't work

Reply:
yep